swift-sdk/Internal/IterableHtmlMessageViewController.swift

```diff
@@ -1,105 +1,61 @@
 //
-//  IterableInAppHTMLViewController.swift
-//  new-ios-sdk
 //
-//  Created by Tapash Majumder on 6/7/18.
-//  Copyright © 2018 Iterable. All rights reserved.
+//  Created by Tapash Majumder on 3/11/19.
+//  Copyright © 2019 Iterable. All rights reserved.
 //
 
 import UIKit
 
-enum InAppNotificationType : Int {
+enum IterableMessageLocation : Int {
```

Comment:
It's `Display type` in template UI.. I'm not sure, this one looks a bit generic, but it's up to you.

Reply:
Lets deal with this one later.